### PR TITLE
Clarified North/South, removed anchors, added code highlights

### PR DESCRIPTION
**Pull request by jason-fox**
After discussion with @dcalvoalonso 

With standard English usage, the “bound”  suffix in implies a direction e.g “Homeward bound" means "going **towards** my home”, therefore “Southbound” means "going **towards** the South”
In contrast, the phrase "to the North of” does not imply direction, it implies relative location — you can be to the North of a town, but travelling Southbound - i.e. towards the city.
The main common usage of Northbound/Southbound is found with motorway traffic reports e.g. “the Northbound carriageway is blocked"

Therefore the following usage is correct common English, and the existing documents should be corrected to use it: 

* Traffic **to the South of** the IoT Agent refers to the exchange of information and commands between a device and an IoT Agent (Native protocols)
Traffic **to the North of** the IoT Agent traffic considers the link between the IoT Agent  and the CB.  (NGSI)
* **Southbound** traffic refers to the commands sent from the CB down to the device
* **Northbound** traffic refers to the measurements sent up from the IoT device to the CB
* **Northbound** traffic also refers to the results of commands  sent up from the IoT device to the CB

I have updated the documents  and removed `<a>`  tags and added code highlights for ReadTheDocs whilst updating.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3901 @@
+{
```
Comment (fgalan) [inline package-lock.json]: I undesrstand this file was added unintentionally and should be removed.

Comment (jason-fox) [reply package-lock.json]: Deleted. Fixed in bf189e4

File: docs/operations.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ while the 'Severity' criterium is as follows:
 * **Major** - The system has a problem that degrades the service and must be addressed
 * **Warning** - It is happening something that must be notified
 
-## <a name="errorcode"/> Error naming code
+## Error naming code
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply docs/operations.md]: Fixed b1deab1 - Regenerated ToC

**Comment by jason-fox** [comment]
@dcalvoalonso - I was just going to update the RTD docs folder, but it probably makes sense to align the files you suggested as well as documentation in the broader sense. - Fixed b1deab1

**Pull request merged**